src/bot.ts

```diff
@@ -5,8 +5,15 @@ import TelegramBot, { Message, User } from 'node-telegram-bot-api'
 import Logger from 'helpers/Logger'
 import { logUnhandledErrors, onShutdown } from 'helpers'
 import { dfusionService } from 'services'
-import { formatAmount } from 'utils/format'
+import { formatAmount, formatAmountFull } from 'utils/format'
 import BN from 'bn.js'
+import { FEE_DENOMINATOR } from 'const'
+
+const WEB_BASE_URL = process.env.WEB_BASE_URL
+assert(WEB_BASE_URL, 'WEB_BASE_URL is required')
```

Comment:
Will probably need the new `assert` @W3stside created on `dex-react`, correct?

Reply:
If it's strict mode 3.7.2, yes

Reply:
Changed the import: 012918a9716ab5bc21a04a0a18db0f1aa70734b0